src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java

```diff
@@ -119,37 +123,4 @@
     return ResponseEntity.ok().body(body);
   }
 
-  @GetMapping(value = "/api/member/courses", produces = "application/json")
-  public ResponseEntity<String> getMyCourses(@RequestHeader("Authorization") String authorization) throws JsonProcessingException {
-    if (authControllerAdvice.getIsAdmin(authorization)){
-      List<Course> courseList = courseRepository.findAll();
-      if(courseList.isEmpty()){
-        return ResponseEntity.notFound().build();
-      }
-      ObjectMapper mapper = new ObjectMapper();
-      String body = mapper.writeValueAsString(courseList);
-      return ResponseEntity.ok().body(body);
-    } 
-    else {
-      List<Course> courseList = courseRepository.findAllByInstructorEmail(authControllerAdvice.getUser(authorization).getEmail());
-      if(courseList.isEmpty()){
-        return getUnauthorizedResponse("instructor");
-      }
-      ObjectMapper mapper = new ObjectMapper();
-      String body = mapper.writeValueAsString(courseList);
-      return ResponseEntity.ok().body(body);
-    }  
-  }
-
-  @GetMapping(value = "/api/member/courses/forInstructor/{email}", produces = "application/json")
-  public ResponseEntity<String> getMyCourses(@RequestHeader("Authorization") String authorization, @PathVariable("email") String email) throws JsonProcessingException {
-    if (!authControllerAdvice.getIsMember(authorization)){
-      return getUnauthorizedResponse("member");
-    }
-    List<Course> courseList = courseRepository.findAllByInstructorEmail(email);
-    ObjectMapper mapper = new ObjectMapper();
-    String body = mapper.writeValueAsString(courseList);
-    return ResponseEntity.ok().body(body);
-  }
-
```

Comment:
It looks like you're deleting work that was put in by another team. We shouldn't be undoing other teams' work, so can you please reinstate this?

Reply:
I added them back in